### PR TITLE
Update install guides - listening on all IP addresses should be an optional step

### PR DESCRIPTION
**Pull request by jasonblais**
Will need to update all relevant docs, just starting a WIP based on the comment here https://community-daily.mattermost.com/core/pl/5cortxstzf8zuqwya5us63meia

### CODE REVIEW

**Comment by jasonblais** [comment]
Once ready, will request appropriate reviews from dev, PM and editors.

File: source/install/install-ubuntu-1604-postgresql.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Assume that the IP address of this server is 10.10.10.1
 
   ``exit``
 
-9. Allow PostgreSQL to listen on all assigned IP Addresses. Open ``/etc/postgresql/9.5/main/postgresql.conf`` as root in a text editor.
+9. (Optional) If you use a different server for your database and the Mattermost app server, and have proper firewall configuration, you may allow PostgreSQL to listen on all assigned IP Addresses. To do so, open ``/etc/postgresql/9.5/main/postgresql.conf`` as root in a text editor.
```
Comment (DSchalla) [inline source/install/install-ubuntu-1604-postgresql.rst]:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost app server, you may allow PostgreSQL to listen on all assigned IP Addresses. To do so, open ``/etc/postgresql/9.5/main/postgresql.conf`` as root in a text editor. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Comment (DSchalla) [inline source/install/install-ubuntu-1604-postgresql.rst]:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost app server, you may allow PostgreSQL to listen on all assigned IP Addresses. To do so, open ``/etc/postgresql/9.5/main/postgresql.conf`` as root in a text editor. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Comment (jasonblais) [reply source/install/install-ubuntu-1604-postgresql.rst]: @DSchalla re-reading this note with fresh eyes, my first question as an admin is: should I allow PostgreSQL to listen on all assigned IP Addresses? When/why should I allow this?

Wondering if you have any insight.

Comment (jasonblais) [reply source/install/install-ubuntu-1604-postgresql.rst]: friendly nudge @DSchalla for next week :)

Comment (DSchalla) [reply source/install/install-ubuntu-1604-postgresql.rst]: @jasonblais Yes, when you are having a dedicated database server and a separate Mattermost server, you'll need to do that.

**Comment by jasonblais** [comment]
Resubmitted here https://github.com/mattermost/docs/pull/2599

**Pull request closed without merging**